[francetv] "No video formats found!" on some videos from france.tv

### Checklist

- [x] I'm reporting that yt-dlp is broken on a **supported** site
- [x] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [x] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [x] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [x] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766), [the FAQ](https://github.com/yt-dlp/yt-dlp/wiki/FAQ), and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=is%3Aissue%20-label%3Aspam%20%20) for similar issues **including closed ones**. DO NOT post duplicates
- [x] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

France

### Provide a description that is worded well enough to be understood

Hello,
I get an error while trying to download some videos from the site `https://www.france.tv/` .
It says (in French) : "Access to this video is incompatible with your configuration. To access it, please update your application."
This happens only with some of the videos (I enclose the result with `https://www.france.tv/films/films-suspense/7160465-sa-majeste-des-mouches.html`), whereas others work just fine (for instance `https://www.france.tv/films/7159211-little-big-man.html`).
I can watch the video on Chrome. The site asks for a login, but it does it too for the video that works with yt-dlp, so this should not be the issue.
I updated to nightly for a try, to no avail.
Thank you for your help.

### Provide verbose output that clearly demonstrates the problem

- [x] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [x] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.france.tv/films/films-suspense/7160465-sa-majeste-des-mouches.html']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2025.05.27.232941 from yt-dlp/yt-dlp-nightly-builds [cc749a8a3] (win_exe)
[debug] Python 3.10.11 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1t  7 Feb 2023)
[debug] exe versions: ffmpeg N-111652-gbf9f6a5e55-20230730 (setts), ffprobe N-111652-gbf9f6a5e55-20230730
[debug] Optional libraries: Cryptodome-3.23.0, brotli-1.1.0, certifi-2025.04.26, curl_cffi-0.10.0, mutagen-1.47.0, requests-2.32.3, sqlite3-3.40.1, urllib3-2.4.0, websockets-15.0.1
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Plugin directories: none
[debug] Loaded 1860 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-nightly-builds/releases/latest
Latest version: nightly@2025.05.27.232941 from yt-dlp/yt-dlp-nightly-builds
yt-dlp is up to date (nightly@2025.05.27.232941 from yt-dlp/yt-dlp-nightly-builds)
[francetv:site] Extracting URL: https://www.france.tv/films/films-suspense/7160465-sa-majeste-des-mouches.html
[francetv:site] 7160465-sa-majeste-des-mouches: Downloading webpage
[francetv] Extracting URL: francetv:bff4f628-5069-4a1a-bd68-06f2b66fe84a#__youtubedl_smuggle=%7B%22hostname%22%3A+%22www.france.tv%22%7D
[francetv] bff4f628-5069-4a1a-bd68-06f2b66fe84a: Downloading desktop chrome video JSON
WARNING: [francetv] francetv said: 2019 "L'accès à cette vidéo est incompatible avec votre configuration.

Pour y accéder, veuillez mettre à jour votre application."
[francetv] bff4f628-5069-4a1a-bd68-06f2b66fe84a: Downloading mobile safari video JSON
WARNING: [francetv] francetv said: 2019 "L'accès à cette vidéo est incompatible avec votre configuration.

Pour y accéder, veuillez mettre à jour votre application."
[debug] Extractor gave empty title. Creating a generic title
ERROR: [francetv] bff4f628-5069-4a1a-bd68-06f2b66fe84a: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1662, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1818, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1877, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2884, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 1151, in raise_no_formats
yt_dlp.utils.ExtractorError: [francetv] bff4f628-5069-4a1a-bd68-06f2b66fe84a: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

hello i have this issue..
2025-06-20 20:31:54 [INFO]  Starting playback for: Perugia - IV November Square
2025-06-20 20:31:54 [INFO]  URL: https://www.skylinewebcams.com/en/webcam/italia/umbria/perugia/piazza-iv-novembre.html
2025-06-20 20:31:54 [INFO]  URL appears to be a web page: https://www.skylinewebcams.com/en/webcam/italia/umbria/perugia/piazza-iv-novembre.html
2025-06-20 20:31:54 [INFO]  Using scraper for web page source
2025-06-20 20:31:54 [INFO]  Entering __init__
2025-06-20 20:31:54 [INFO]  Entering get_stream_url
2025-06-20 20:31:54 [INFO]  Entering fetch for URL: https://www.skylinewebcams.com/en/webcam/italia/umbria/perugia/piazza-iv-novembre.html
2025-06-20 20:31:54 [INFO]  Fetching URL: https://www.skylinewebcams.com/en/webcam/italia/umbria/perugia/piazza-iv-novembre.html
2025-06-20 20:31:54 [INFO]  Found YouTube video ID: 8TZ8YRt9nYc
2025-06-20 20:31:54 [INFO]  Using stream URL: https://www.youtube.com/watch?v=8TZ8YRt9nYc
2025-06-20 20:31:54 [INFO]  Playing YouTube: https://www.youtube.com/watch?v=8TZ8YRt9nYc
2025-06-20 20:31:54 [INFO]  Title: Perugia - IV November Square
2025-06-20 20:31:54 [INFO]  Using system-wide yt_dlp
2025-06-20 20:31:55 [DEBUG] [youtube] Extracting URL: https://www.youtube.com/watch?v=8TZ8YRt9nYc
2025-06-20 20:31:55 [DEBUG] [youtube] 8TZ8YRt9nYc: Downloading webpage
2025-06-20 20:31:56 [DEBUG] [youtube] 8TZ8YRt9nYc: Downloading android player API JSON
2025-06-20 20:31:57 [ERROR] [0;31mERROR:[0m [youtube] 8TZ8YRt9nYc: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. 
2025-06-20 20:31:57 [ERROR] yt-dlp failed: [0;31mERROR:[0m [youtube] 8TZ8YRt9nYc: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. 
i use enigma2 on box and major video are ok.. but some have this error

Thank's

--- a/yt_dlp/extractor/francetv.py
+++ b/yt_dlp/extractor/francetv.py
@@ -124,9 +124,10 @@ class FranceTVIE(InfoExtractor):
             elif code := traverse_obj(dinfo, ('code', {int})):
                 if code == 2009:
                     self.raise_geo_restricted(countries=self._GEO_COUNTRIES)
-                elif code in (2015, 2017):
+                elif code in (2015, 2017, 2019):
                     # 2015: L'accès à cette vidéo est impossible. (DRM-only)
                     # 2017: Cette vidéo n'est pas disponible depuis le site web mobile (b/c DRM)
+                    # 2019: L'accès à cette vidéo est incompatible avec votre configuration. (DRM-only)
                     drm_formats = True
                     continue
                 self.report_warning(

